packages/router/package.json

```diff
@@ -14,7 +14,7 @@
 	"dependencies": {
 		"@ionic/core": "^8.1.0",
 		"solid-element": "^1.8.0",
-		"@ionic-solidjs/core": "1.0.2"
+		"@ionic-solidjs/core": "workspace:^*"
```

Comment:
Wouldn't bun already install from workspace here, btw? release-please will revert this eventually.

Reply:
I can test it but if it does not have `workspace:` "anotation" (or what it is), it should take it from NPM registry. At least other package managers work like that.
`workspace:` anotation should be replaced by the package manager while publish (with no effect on original package.json)

Reply:
`pnpm` will install from workspace if the version range is matching; I'll merge this but the workspace link _will_ auto-revert on next publish.